03_dagster/definitions.py

```
@@ -17,7 +17,8 @@
 import time
 import pandas as pd
 import duckdb
-from dagster import asset, AssetExecutionContext
+import subprocess
+from dagster import asset, AssetExecutionContext, AssetSelection, define_asset_job, op, In
```

Ruff (0.12.2)

`dagster.op` imported but unused

Remove unused import

(F401)